core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -273,10 +273,10 @@ class HttpGoogleServicesDAO(
 
     retryWithRecoverWhen500orGoogleError(() => {
       val bucket = new Bucket().setName(bucketName)
-      val deleteAfterOneYear = new Lifecycle.Rule()
+      val storageLogExpiration = new Lifecycle.Rule()
```

Comment:
Ah! So misleading is bad, but I might rename it to `deleteExpired` or `deleteAfterExpiration`. Totally subjective (no need to change), but to me a rule feels more like a verb than a noun.